proposals/drafts/elections.rst

```diff
@@ -43,46 +255,502 @@ extras
     Common to all Open Civic Data types, the value is a key-value store suitable for storing arbitrary information not covered elsewhere.
 
 
+BallotMeasureContest
+--------------------
+
+A subclass of ``ContestBase`` for representing a ballot measure before the voters, including summary statements on each side.
+
+id
+    Open Civic Data-style id in the format ``ocd-ballotmeasurecontest/{{uuid}}``
+
+con_statement
+    **optional**
+    Specifies a statement in opposition to the ballot measure. It does not necessarily appear on the ballot (string).
+
+effect_of_abstain
+    **optional**
+    Specifies the effect abstaining from voting on the ballot measure, i.e., whether abstaining is considered a vote against it (string).
+
+full_text
+    **optional**
+    Specifies the full text of the ballot measure as it appears on the ballot (string).
+
+info_uri
+    **optional**
+    Specifies a URI that links to additional information about the ballot measure (string).
+
+passage_threshold
+    **optional**
+    Specifies the threshold of votes the ballot measure needs in order to pass. The default is a simple majority, i.e., "50% plus one vote". Other common thresholds are "three-fifths" and "two-thirds".
+
+pro_statement
+    **optional**
+    Specifies a statement in favor of the referendum. It does not necessarily appear on the ballot (string).
+
+summary_text
+    **optional**
+    Specifies a short summary of the ballot measure that is on the ballot, below the title, but above the text.
+
+type
+    **optional**
+    Enumerated among "ballot-measure", "referendum", "initiative" and "other".
+
+other_type
+    **optional**
+    Allows for cataloging a new type of ballot measure option, when type is specified as "other" (string).
+
 
 CandidateContest
 ----------------
 
+A subclass of ``ContestBase`` for repesenting a contest among candidates competing for election to a public office.
+
 id
-    ocd identifier
+    Open Civic Data-style id in the format ``ocd-candidatecontest/{{uuid}}``.
 
-election_id
-    reference to the ``Election`` this contest is part of
+filing_deadline
+    **optional**
+    Specifies the date and time when a candidate must have filed for the contest for the office (datetime).
+
+is_runoff
+    Indicates a contest to decide a prior contest that ended with no candidate receiving a majority of the votes (boolean).
```

Comment:
Interesting idea. Would this just be something like and optional `CandidateContest.runoff_for_contest_id` and, if we added that, would we still need `is_runoff`?